<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolutionParser</class>
 <widget class="QWidget" name="SolutionParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>393</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI Semilight</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SolutionParser</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="solutionLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="tabStopWidth">
    <number>27</number>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Comments (C)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="copyUrl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy URL</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/copy-icon-25.png</normaloff>../../Desktop/copy-icon-25.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="downloadProgress">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBaseButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Update DB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Download All Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCustomButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add  Ð¡ustom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
